albumentations/augmentations/transforms.py

```diff
@@ -3107,3 +3108,40 @@ def apply(self, img, transforms=(), **params):
 
     def get_transform_init_args_names(self):
         return ("brightness", "contrast", "saturation", "hue")
+
+
+class Sharpen(ImageOnlyTransform):
+    """Sharpen the input image and overlays the result with the original image.
+
+    Args:
+        alpha ((float, float)): range to choose the visibility of the sharpened image. At 0, only the original image is
+            visible, at 1.0 only its sharpened version is visible. Default: (0.2, 0.5).
+        lightness ((float, float)): range to choose the lightness of the sharpened image. Default: (0.5, 1.0).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+    """
+
+    def __init__(self, alpha=(0.2, 0.5), lightness=(0.5, 1.0), always_apply=False, p=0.5):
+        super(Sharpen, self).__init__(always_apply, p)
+        self.alpha = to_tuple(alpha, 0.0)
+        self.lightness = to_tuple(lightness, 0.0)
+        self._sharpening_matrix = self._generate_sharpening_matrix()
+
+    def _generate_sharpening_matrix(self):
+        alpha_sample = random.uniform(*self.alpha)
+        if not 0 <= alpha_sample <= 1.0:
+            raise ValueError(f"Expected 'alpha' to be in the interval [0.0, 1.0], got {alpha_sample}")
```

Comment:
Better to move checking borders to `__init__`

Reply:
Same as in Emboss, _sharpening_matrix should be generated for each new call (see BrightnessContrastAdjust for instance)

Reply:
fixed both